### ISSUE TITLE
Display problem in Conky

### ISSUE THREAD

**Issue by GabenCrying** [chubin/wttr.in#555]
This may belong strictly under conkys github but I was curious if anyone had a cure here. I appreciate your guys' service so much. It is beautiful and I would very much like to simply run it as a background conky. However, it looks like the following.

![g](https://user-images.githubusercontent.com/73147914/102800329-b6b3a200-4381-11eb-990c-393cef2d6f4b.png)

What am I missing here?

**Comment by GabenCrying** [comment]
I have tried using the image format, however conky fails to load image without reason each time. Any advice is sincerely appreciated. If I am not simply missing a font or something simply, then I will take this to conkys github. Thank you

**Comment by GabenCrying** [comment]
Bumper!

**Comment by gregdan3** [comment]
https://unix.stackexchange.com/questions/230613/how-can-i-display-ansi-color-in-a-cli-conky-display

This looks to be a potential solution. Conky doesn't seem to have base support for ANSI colors, which wttr.in uses, and instead has its own color syntax. You need to convert between the two.

**Comment by chubin** [comment]
@GabenCrying Sorry for the late reply.

As far as I know, all of our onky users use the image-based format.
or one-line based format (neither ANSI, nor HTML).

That's how it looks like:

![o7S38fav_o](https://user-images.githubusercontent.com/3875145/103422483-0aa06100-4ba2-11eb-8527-11038ab45287.png)

What URL do you use to load the image?

**Comment by chubin** [comment]
@gregdan3 Gregory, I think that we simply need to add a new output format to wttr.in
(additionally to ANSI, PNG, HTML, JSON, Prometheus, and XML).
Then the query would look like:

`wttr.in/London?format=conky`

What do you think?

**Comment by zcake3000** [comment]
you can disable ANSI colors in wttr.in by using T flag(options).

**Comment by chubin** [comment]
@zcake3000 Yes, but I think that the idea is exactly keep the colors, and not disable them, because it looks much better with colors
